.github/workflows/cypress.yml

```diff
@@ -1,14 +1,9 @@
 name: Mocked backend tests
 
-on:
-  push:
-    branches: [master, develop]
-  pull_request:
-    branches: [master, develop]
+on: push
```

Comment:
This is on push to every branch ever. Probably want this as:
```
  push:
    branches: [master, develop]
  pull_request:
    branches: [master, develop]
```

to mirror the behaiour of testing in console :+1:

Reply:
Basically always run on push to dev and master and run on push to every branch with a PR open, as this will run on every commit which is a bit 😬

Reply:
@dexturr that looks good, @Ditmir-Vega please could you make it so?